Incorrect SupportedOSPlatform attributes on various properties

Similar bug report as here: https://github.com/xamarin/xamarin-macios/issues/19066

### Steps to Reproduce

Use `NSMutableParagraphStyle.TextLists` in a project with MinimumOSVersion < 16.0

### Expected Behavior
No warning

### Actual Behavior
`/Users/tipa/GitHub/diarium/Diarium/Classes/Helper_macios.cs(99,25): warning CA1416: This call site is reachable on: 'iOS' 15.0 and later, 'maccatalyst' 15.0 and later. 'NSParagraphStyle.TextLists' is only supported on: 'ios' 16.0 and later, 'maccatalyst' 16.0 and later, 'tvos' 16.0 and later. (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1416) 
`

### Comments

Documented to be available since iOS 7:
https://developer.apple.com/documentation/uikit/nsparagraphstyle/1534193-textlists?language=objc

https://github.com/xamarin/xamarin-macios/blob/343699204a3ff1f591e861e046baa672fc7e6626/src/xkit.cs#L1797-L1799
I haven't tested functionality yet, but at least setting & getting the property does not crash on iOS 15:

```
var xx = new NSMutableParagraphStyle();
xx.TextLists = new[] { new NSTextList(NSTextListMarkerFormats.Circle) };
var yy = xx.TextLists[0];
```


Documented to be available since iOS 9:
https://developer.apple.com/documentation/uikit/nsparagraphstyle/3667463-linebreakstrategy?language=objc

https://github.com/xamarin/xamarin-macios/blob/343699204a3ff1f591e861e046baa672fc7e6626/src/xkit.cs#L1811-L1814

There's probably even more properties for which Apple changed their min API level with the last SDK...

### Environment
.NET 8 RC2

--- a/src/xkit.cs
+++ b/src/xkit.cs
@@ -1862,7 +1862,6 @@ namespace UIKit {
 #endif
 
 		[NoWatch, MacCatalyst (13, 1)]
-		[NoMacCatalyst]
 		[Override]
 		[Export ("textLists")]
 		NSTextList [] TextLists { get; set; }

--- a/src/xkit.cs
+++ b/src/xkit.cs
@@ -244,15 +244,16 @@ namespace UIKit {
 		GreaterThanOrEqual = 1,
 	}
 
-	[Watch (7, 0), TV (14, 0), iOS (14, 0)]
-	[Mac (11, 0)]
 	[MacCatalyst (13, 1)]
 	[Flags]
 	[Native]
 	public enum NSLineBreakStrategy : ulong {
+		[Mac (11, 0), iOS (14, 0), TV (14, 0), Watch (7, 0), MacCatalyst (14, 0)]
 		None = 0x0,
 		PushOut = 1uL << 0,
+		[Mac (11, 0), iOS (14, 0), TV (14, 0), Watch (7, 0), MacCatalyst (14, 0)]
 		HangulWordPriority = 1uL << 1,
+		[Mac (11, 0), iOS (14, 0), TV (14, 0), Watch (7, 0), MacCatalyst (14, 0)]
 		Standard = 0xffff,
 	}
 
@@ -1745,7 +1746,7 @@ namespace UIKit {
 		NSTextTableBlock [] TextBlocks { get; [NotImplemented] set; }
 #endif
 
-		[iOS (16, 0), TV (16, 0), NoWatch, MacCatalyst (16, 0)]
+		[NoWatch, MacCatalyst (13, 1)]
 		[Export ("textLists")]
 		NSTextList [] TextLists { get; [NotImplemented] set; }
 
@@ -1759,8 +1760,7 @@ namespace UIKit {
 		[Export ("headerLevel")]
 		nint HeaderLevel { get; [NotImplemented] set; }
 
-		[Mac (11, 0), Watch (7, 0), TV (14, 0), iOS (14, 0)]
-		[MacCatalyst (14, 0)]
+		[MacCatalyst (13, 1)]
 		[Export ("lineBreakStrategy")]
 		NSLineBreakStrategy LineBreakStrategy { get; [NotImplemented] set; }
 	}
@@ -1861,7 +1861,7 @@ namespace UIKit {
 		NSTextTableBlock [] TextBlocks { get; set; }
 #endif
 
-		[iOS (16, 0), TV (16, 0), NoWatch, MacCatalyst (16, 0)]
+		[NoWatch, MacCatalyst (13, 1)]
 		[NoMacCatalyst]
 		[Override]
 		[Export ("textLists")]
@@ -1879,8 +1879,7 @@ namespace UIKit {
 		[Override]
 		nint HeaderLevel { get; set; }
 
-		[Mac (11, 0), Watch (7, 0), TV (14, 0), iOS (14, 0)]
-		[MacCatalyst (14, 0)]
+		[MacCatalyst (13, 1)]
 		[Override]
 		[Export ("lineBreakStrategy", ArgumentSemantic.Assign)]
 		NSLineBreakStrategy LineBreakStrategy { get; set; }

